app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) != 3:
+        raise ValueError("Missed some arguments.")
+    cp, file_in, file_out = command.split()
+    if cp != "cp":
+        raise ValueError("Incorrect command.")
+    if file_in == file_out:
+        raise ValueError("Incorrect FileIn and FileOut.")
```

Comment:
u don't need to do anything if files are the same
https://github.com/mate-academy/py-copy-file#:~:text=It%20must%20do%20nothing%20in%20case%20the%20user%20is%20trying%20to%20copy%20file%20to%20file%20with%20the%20same%20name.